web/source/kmwdebug.js

```diff
@@ -20,8 +20,6 @@ if(!window['keyman']['initialized']) {
     {
       var dhost = document.createElement('DIV');
       dhost.style.display = 'block';
-      if(!keymanweb._IE)
-        dhost.style.position = 'fixed';
```

Comment:
Is this right? You've deleted code that's `!_IE`?

Reply:
Oops.  I probably played it a little too fast and loose on this particular file; I've never once actually used its code.  It's just kind of... lingered on, so far as I know.